app/enterprise/changelog.md

```diff
@@ -2,6 +2,142 @@
+  - `user_token` must now be provided when creating a new RBAC user (in a POST request to rbac/users) and is no longer auto-generated if left blank
+
+**FEATURES**
+- **Kong Manager**
+  - User information is no longer stored in local storage. A user exchanges credentials for a session. See the Session Plugin for details.
+  - Enable and view plugins in context of a route and a service
+  - Type-ahead service name/ID search on routes form
+  - Easily scan/copy IDs in service table
+  - Contextual help and error messages for forms
+  - Breadcrumbs to more easily tell where in app you are, navigate/pivot to other pages
+  - In-context doc links
+  - Global navigation links to docs and support portal
+  - Improved Info tab for cluster & config info
+  - Detailed debug tracing can be enabled which outputs information about various portions of the request lifecycle, such as DB or DNS queries, plugin execution, core handler timing, etc
+- **Dev Portal**
+  - Developers can now sign up to multiple workspaces with the same email address.
+  - Developer Portal is disabled for each workspace by default.
+  - Default theme shipped with Kong Enterprise now supports IE11.
+  - User information is no longer stored in local storage. A user exchanges credentials for a session. See the Session Plugin for details.
+  - Developer Portal page, partial, and specification look-ups moved to server-side.
+  - Added support for OAS3 body parameters.
+  - Added CORS header configuration option to workspace portal configuration object to allow explicit CORS configuration.
+  - Added a live editor built on VSCode core for improved customization of Developer Portal within Kong Manager
+  - Added custom registration fields support and custom registration field interface.
+  - Added methods to links in the sidebar on documentation in the default portal theme.
+  - Added hover states to drop-downs in the sidebar of documentation in the default portal theme.
+  - Added a right border to the sidebar in the default portal theme.
+  - Changed "Full Name" registration field to be a custom registration field, and can be managed through the custom fields -   interface in Kong Manager.
+  - Changed custom checkbox to native checkbox in the default developer portal theme on the dashboard.
+- **Core**
+  - New RBAC user tokens are not stored in plaintext. If upgrading to this version, any existing tokens will remain in plaintext until either a) the token is used in an Admin API call or b) the rbac_user record is PATCHed (even if the PATCH request includes the existing value of `user_token`).
+- **Plugins**
+  - **Response Transformer Advanced**: 
+    - Conditional transformations on response status through the new flag if_status, a subfield in all transformations (e.g., `add.if_status`, `remove.if_status`); the transformation only happens if the response code matches one of the codes in the `if_status field
```

Comment:
Missing closing backtick in if_status

Reply:
Could we somehow explicitly highlight which plugins are introduced in this release? Resp Transformer Advanced and Request Validator are two such examples.

Reply:
Sure, maybe I'll put _new_ next to them? @levfishbluefish , thoughts?

Reply:
```suggestion
    - Conditional transformations on response status through the new flag if_status, a subfield in all transformations (e.g., `add.if_status`, `remove.if_status`); the transformation only happens if the response code matches one of the codes in the `if_status field`
```

Reply:
**Response Transformer Advanced** _new_